tools/minibrowser/src/main/java/com/wpe/tools/minibrowser/Browser.kt

```diff
@@ -151,6 +151,29 @@ class MainActivity : AppCompatActivity(R.layout.activity_main) {
                     progressView.visibility = View.GONE
                 }
             }
+
+            override fun onShowCustomView(view: View?, callback: WebChromeClient.CustomViewCallback?) {
+                fullscreenView?.let {
+                    (it.parent as ViewGroup).removeView(it)
+                }
+                fullscreenView = view
+                window.addFlags( WindowManager.LayoutParams.FLAG_FULLSCREEN)
```

Comment:
There is an extra space after the opening (
In Android Studio you can automatically reformat the file by pressing Ctrl+Alt+Enter

Reply:
fixed